apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -307,7 +308,13 @@ export class RushConfiguration {
    * an RushConfiguration object.
    */
   public static loadFromConfigurationFile(rushJsonFilename: string): RushConfiguration {
-    const resolvedRushJsonFilename: string = path.resolve(rushJsonFilename);
+    let resolvedRushJsonFilename: string = path.resolve(rushJsonFilename);
+    try {
+      resolvedRushJsonFilename = trueCasePathSync(resolvedRushJsonFilename);
+    } catch (error) {
+      throw new Error(`Unable to resolve rush.json filename. Inner error: ${error}`);
+    }
+
     const rushConfigurationJson: IRushConfigurationJson = JsonFile.load(resolvedRushJsonFilename);
```

Comment:
Should `JsonFile.load()` come first?

Reply:
Or rather, in the simple case where the file is missing, what does the error message look like?  Will this "Inner error" stuff look weird?